qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -229,37 +229,29 @@ set_helm_params() {
 }
 
 set_uaa_sizing_params() {
-    return 0 # Sizing UAA up breaks CATS
     if [[ "${HA}" == true ]]; then
-        if semver_is_gte "$(helm_chart_version)" 2.11.0; then
+        if [[ ${SCALED_HA} == true ]]; then
+            HELM_PARAMS+=(--set=sizing.{uaa,mysql}.count=2)
+        else
             # HA UAA not supported prior to 2.11.0
             HELM_PARAMS+=(--set=config.HA=true)
```

Comment:
We don't want this at all for UAA right now, right?

Reply:
OK, based on discussion, we need separate flags for custom UAA sizing and custom SCF sizing (with better names than `SCALED_HA`).

Proposed `CUSTOM_UAA_SIZING` and `CUSTOM_SCF_SIZING`

Because, currently on this branch, `SCALED_HA=true` won't be triggered on the next release for pre-upgrade deploys, we would unintentionally set `config.HA=true` for UAA in testing after the next release. Instead, I've recommended that `CUSTOM_UAA_SIZING=true` be set independent of [whether the deploy detects it's a pre-upgrade deploy requiring the pxc workaround](https://github.com/SUSE/cf-ci/blob/ffe344c2f46c976ad949a0ba0d11bec7a78cde53/qa-pipelines/tasks/cf-deploy.sh#L18) (as that will be unnecessary after the 1.5 release